modernizer-annotation-processor/src/main/java/org/gaul/annotation_processsor/ModernizerAnnotationProcessor.java

```diff
@@ -53,36 +57,56 @@ public final boolean process(
         RoundEnvironment roundEnv
     ) {
         for (TypeElement annotation : annotations) {
-            List<String> annotatedClasses =
-                getAnnotatedClasses(roundEnv, annotation);
-            makeAnnotatedClassesFile(annotatedClasses);
+            List<String> annotatedClasses = new ArrayList<String>();
+            List<String> annotatedMethods = new ArrayList<String>();
+            getAnnotatedElements(
+                roundEnv, annotation, annotatedClasses, annotatedMethods);
+            makeAnnotatedElementsFiles(annotatedClasses, annotatedMethods);
         }
         return true;
     }
 
-    private List<String> getAnnotatedClasses(
+    private void getAnnotatedElements(
         RoundEnvironment roundEnv,
-        TypeElement annotation
+        TypeElement annotation,
+        List<String> annotatedClasses,
+        List<String> annotatedMethods
     ) {
-        List<String> annotatedClasses = new ArrayList<String>();
         for (Element element : roundEnv.getElementsAnnotatedWith(annotation)) {
             List<String> warnings = Arrays.asList(
                     element.getAnnotation(SuppressWarnings.class).value());
             if (warnings.contains("modernizer")) {
                 if (element.getKind().isClass()) {
                     annotatedClasses.add(getClassHeader(element));
+                } else if (element.getKind().toString().equals("METHOD") ||
+                    element.getKind().toString().equals("CONSTRUCTOR")) {
```

Comment:
Rather than using `toString()` and string comparison, could we do an equality check on the enum?